polymetis/tests/scripts/0_joint_positions.py

```diff
@@ -65,7 +65,7 @@ def test_new_joint_pos(robot, joint_pos_desired):
     state_log = robot.go_home(time_to_go=time_to_go)
     time.sleep(0.5)
 
-    joint_pos = test_new_joint_pos(robot, joint_pos_desired, time_to_go=time_to_go)
+    joint_pos = test_new_joint_pos(robot, joint_pos_desired)
```

Comment:
As I said in previous PR, not quite sure why some are 3.0 and some are 1.0

Reply:
Mentioned in the previous PR. This line of change removes a bug as `test_new_joint_pos` is not supposed to have a time_to_go argument. I debugged it in both PRs but weird that it shows up here.